packages/box/lib/utils/unbox.ts

```diff
@@ -25,9 +25,11 @@ async function verifyVCSURL(url: string) {
       .replace("github.com", "raw.githubusercontent.com")
       .replace(/#.*/, "")}/master/truffle-box.json`
   );
+
+  const testUrl = `https://${configURL.host}${configURL.path}`;
   try {
     await axios.head(
-      `https://${configURL.host}${configURL.path}`,
+      testUrl,
```

Comment:
Should we name this something more like `repoUrl`? Not necessary but just an idea, it seems a little more explicit.

Reply:
Yes, this is clearer.